### PR TITLE
Add README.md notice about update interval changes and issues

### PR DESCRIPTION
**Pull request by DurgNomis-drol**
Please give feedback 😃

### CODE REVIEW

**Comment by DurgNomis-drol** [comment]
Also this means that people need to update their translations to include the new string. 😃

File: README.md [hunk 1]
```diff
@@ -47,6 +47,16 @@ This is a custom component that is emerging from the
 local google assistant device (like Google Home/Nest etc) authentication
 tokens and use those tokens making API calls to Google Home devices.
 
+## IMPORTANT!!
```
Comment (KapJI) [inline README.md:50]:
```suggestion
## IMPORTANT
```

File: README.md [hunk 2]
```diff
@@ -47,6 +47,16 @@ This is a custom component that is emerging from the
...
+
+With version 1.6 we have changed the update interval from 10 seconds to 180 seconds.
+This is due to this [#202](https://github.com/leikoilja/ha-google-home/issues/202) issue. This because of some people
+experincing devices timing out when trying to get data from them.
```
Comment (KapJI) [inline README.md:54]:
```suggestion
This is due to [#202](https://github.com/leikoilja/ha-google-home/issues/202). Some people are
experiencing devices timing out when trying to fetch data from them.
```

File: README.md [hunk 3]
```diff
@@ -47,6 +47,16 @@ This is a custom component that is emerging from the
...
+
+It is possible to manually change the update interval by pressing the `configure` button on the integration
+page. Default is 180 seconds. Change this at your own risk!. If your devices are timing out,
```
Comment (KapJI) [inline README.md:57]:
```suggestion
page. Default is 180 seconds. Change this at your own risk! If your devices are timing out,
```

File: README.md [hunk 4]
```diff
@@ -47,6 +47,16 @@ This is a custom component that is emerging from the
 local google assistant device (like Google Home/Nest etc) authentication
 tokens and use those tokens making API calls to Google Home devices.
 
+## IMPORTANT!!
+
+With version 1.6 we have changed the update interval from 10 seconds to 180 seconds.
+This is due to this [#202](https://github.com/leikoilja/ha-google-home/issues/202) issue. This because of some people
+experincing devices timing out when trying to get data from them.
+
+It is possible to manually change the update interval by pressing the `configure` button on the integration
```
Comment (KapJI) [inline README.md:56]:
```suggestion
You can manually change the update interval by pressing the `configure` button on the integration
```

**Review by KapJI — CHANGES_REQUESTED** [review]

**Comment by KapJI** [comment]
> Also this means that people need to update their translations to include the new string. 😃

Can you update them? Just using Google Translate should be sufficient.

**Comment by DurgNomis-drol** [comment]
@KapJI Wouldn't that be better for separate PR?

**Comment by KapJI** [comment]
I don't mind either way.

**Review by KapJI — APPROVED** [review]

**Comment by leikoilja** [comment]
Awesome , thanks guys! 🚀

**Pull request merged**